<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EventsCreateWindow</class>
 <widget class="QDialog" name="EventsCreateWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>330</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>330</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="CreateEventBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>430</width>
     <height>310</height>
    </rect>
   </property>
   <property name="title">
    <string>Create Event</string>
   </property>
   <widget class="QGroupBox" name="TitleBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string>Title</string>
    </property>
    <widget class="QPlainTextEdit" name="TitlePlainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="DescriptionBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>260</width>
      <height>230</height>
     </rect>
    </property>
    <property name="title">
     <string>Description</string>
    </property>
    <widget class="QPlainTextEdit" name="DescriptionPlainTextEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>240</width>
       <height>200</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="EventDateBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string>Event Date</string>
    </property>
    <widget class="QDateEdit" name="EventDateEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="StartTimeBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string>Start Time</string>
    </property>
    <widget class="QTimeEdit" name="StartTimeEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="EndTimeBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>120</width>
      <height>50</height>
     </rect>
    </property>
    <property name="title">
     <string>End Time</string>
    </property>
    <widget class="QTimeEdit" name="EndTimeEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>390</width>
      <height>40</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="HorizontalLayout">
     <item>
      <spacer name="HorizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="EventCreateButton">
       <property name="text">
        <string>Create Event</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
